iceoryx2-pal/posix/src/windows/constants.rs

```diff
@@ -15,6 +15,9 @@
 
 use crate::posix::types::*;
 
+pub const GLOBAL_CONFIG_PATH: &str = "C:/";
```

Comment:
What do you think about using `C:/ProgramData`? Classic iceoryx is also using that folder for the default lookup since there were issues with `C:/`